packages/contentstack-audit/src/messages/index.ts

```diff
@@ -77,7 +77,14 @@ function $t(msg: string, args: Record<string, string>): string {
 
     for (const key of Object.keys(args)) {
       const escapedKey = escapeRegExp(key);
-      msg = msg.replace(new RegExp(`{${escapedKey}}`, 'g'), escapeRegExp(args[key]) || escapedKey);
+      const escapedKeyRegex = new RegExp(`{${escapedKey}}`, 'g');
+      let { status } = validateRegex(escapedKeyRegex)
+      if (status === 'safe') {
+        const sanitizedValue = args[key] ? escapeRegExp(args[key]) : '';
+        msg = msg.replace(escapedKeyRegex, sanitizedValue || escapedKey);
+      } else {
+
+      }
```

Comment:
@cs-raj, log the error message in else condition

Reply:
Removed the else block